services/webfinger/TODO.md

```diff
@@ -0,0 +1,138 @@
+
+# TODO
```

Comment:
```suggestion
## TODO
```

Reply:
nah, the first heading really is lvl 1

Reply:
hahaha, sorry, it was too close/near of readme.md... 🤣